script/start-proposer.sh

```diff
@@ -29,5 +29,6 @@ fi
 if [ "$ENABLE_PROPOSER" == "true" ]; then
     taiko-client proposer ${ARGS}
 else
+    echo "PROPOSER IS DISABLED"
```

Comment:
this will just echo into the terminal infinitely right?

Reply:
Hey d1onys1us,

I can see how my update might give that impression at first glance.

But no worries, the echo "PROPOSER IS DISABLED" will only run once. It's there to give a quick heads-up about the ENABLE_PROPOSER setting before the script just chills out with that sleep infinity command. Here's a snapshot of how it looks in action:
![proposedisabled](https://github.com/taikoxyz/simple-taiko-node/assets/22820692/dbe82bf1-95ac-46f0-9dbe-6134fab0a845)

I added it so that anyone running the script knows right away why it might seem like it's "doing nothing." Just a small tweak for a bit more clarity.

Hope that clears things up! Let me know if there's anything else. 😊

Reply:
it does. sorry for the late response, and thanks for your contribution :)